src/causalprog/algorithms/moments.py

```diff
@@ -32,7 +32,7 @@ def expectation(
     rng_key: jax.Array,
 ) -> float:
     """Estimate the expectation of a graph."""
-    return sample(graph, outcome_node_label, samples, rng_key=rng_key).mean()
+    return moment(1, graph, outcome_node_label, samples, rng_key=rng_key)
```

Comment:
I can't comment on the line above so leaving a comment nearby, but all the docstrings here refer to the graph as the object we are taking moments/samples/etc from. But in actual fact it's the (RVs attached to the) nodes right?

So 

```python
"""Estimate the expectation of a graph."""
```

should probably be

```python
"""Estimate the expectation of (the RV attached to) a node in the graph."""
```

Similarly for the other functions here.